inheritance/src/main/java/inheritance/Shop.java

```diff
@@ -0,0 +1,29 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Shop {
+    public String name;
+    public String description;
+    public String price;
+    public ArrayList<Review> reviews = new ArrayList<>();
+
+//  Constructor
+    public Shop (String name, String description, String price){
+        this.name = name;
+        this.description = description;
+        this.price = price;
+    }
+
+//  Add Review
+    public void addShopReview(Review review){
```

Comment:
Food for thought: Is there a way to make sure the same review isn't added more than once?